lib/keen/styles/variables.scss

```diff
@@ -33,9 +33,8 @@ $ui-input-height: rem(32px) !default;
 $ui-input-margin-bottom: rem(16px) !default;
 
 // Input label
-$ui-input-label-color: $secondary-text-color !default;
+
 $ui-input-label-color--hover: black !default;
-$ui-input-label-color--active: $brand-primary-color !default;
```

Comment:
`ui-input-border-color--active` is still in use within `lib/KSelect/index.vue` as well